zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1281,6 +1279,10 @@ protected void doRetry() throws Exception {
         });
   }
 
+  public boolean getSyncStatus() {
```

Comment:
I have multiple questions.
1. If this is just for test, please make it package-private.
2. Do we really need to check the read count one by one increase?

Reply:
1/ Will make this package level by removing public.

2/ See my comment https://github.com/apache/helix/pull/1295/files#r474351453. See if this address the concern.

Reply:
Sorry that I cannot make the link work. Maybe because code has been changed. Could you please copy-paste your comment here? I still think this is not necessary.

Reply:
See the discussion above about why think we can't get rid of this getSyncStatus().

Reply:
removed.